.github/workflows/build_and_test.yml

```diff
@@ -241,6 +241,7 @@ jobs:
         restore-keys: |
           ${{ matrix.java }}-${{ matrix.hadoop }}-coursier-
     - name: Free up disk space
+      if: hashFiles('./dev/free_disk_space')
```

Comment:
cc @HyukjinKwon @zhengruifeng @Yikun Is the usage of this expression correct?

Reply:
I think you can just do an if-else as you did below?

Reply:
done ~